### PR TITLE
Getting rid of every muppet using f1

### PR DESCRIPTION
**Pull request by LW001**
TODO: Getting rid of every :report: in the feed when :reported: gets shown, where's that bug pls help

### CODE REVIEW

File: Commands/Commands/queue.js [hunk 1]
```diff
@@ -358,7 +358,10 @@ commands.registerVote = {
                     msg.addReaction({
                       id: '296752137935912960',
                       name: 'f1'
-                    }).then(setTimeout(() => msg.removeReaction({id: '296752137935912960', name: 'f1'}), 2500))
+                    }).then(() => {
+                      var muppetsArr = msg.fetchReactions({id: '296752137935912960', name: 'f1'}, bot.User) // muppetsArr => People trying to be funny by reacting to f1
```
Comment (Dougley) [inline Commands/Commands/queue.js]: Wrong usage of the [`fetchReations()` function](http://qeled.github.io/discordie/#/docs/IMessage?p=IMessage%23fetchReactions&_k=cgymn2)

**Review by Dougley — CHANGES_REQUESTED** [review]

**Comment by Arzte** [comment]
Fix file conflicts please

**Review by nullpixel — DISMISSED** [review]
please resolve all merge conflicts

**Comment by Arzte** [comment]
Fixed merge conflicts

**Comment by Dougley** [comment]
The diff for this PR is empty, if you still want this merged, I'd suggest making a new PR.

**Pull request closed without merging**